ATC/src/inlineStaticHelpers.h

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
This file should have a [header guard](https://www.learncpp.com/cpp-tutorial/header-guards/) to prevent duplicate includes. 

Also, since this file references `Vec3`, there should be an `#include "Plane.h"` at the top of the file. It will be protected from duplicate includes thanks to the header guard in `Plane.h`.

Reply:
I was wondering why I kept getting compile errors with duplicates when I was generously including things... This explains it, thank you. Added in b1d9fb7e3deffaf386c7a3ce2963011804976860